support/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -352,14 +359,269 @@ Fastly keeps previous versions of your configuration, and can show the "diff" be
+
+The steps below are based on [this guide](https://docs.microsoft.com/en-us/azure/frontdoor/quickstart-create-front-door#create-a-front-door-for-your-application) to creating a Front Door instance.
+
+1. From the home page or the Azure menu (top left), select Resource Groups. If you don't have one already, create a Resource Group for your project. Give this a name. Choose a region; this affects only where the Azure project metadata is stored.
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-create-resource-group.png)
+
+1. Select "Review and Create", then "Create". This should return you back to your list of Resource Groups.
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-select-resource-group.png)
+
+1. From the home page or the Azure menu, select Create a resource. Select Networking > See All > Front Door.
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-front-door1.png)
+
+    Choose Create.
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-front-door2.png)
+
+    Select your resource group. Select "Next: Configuration".
+
+1. In Frontends/domains, select `+` to open "Add a frontend host". Give your host a name - this needs to be a valid, unique subdomain of the domain `.azurefd.net`. Choosing a name based on your custom tracking domain should help to ensure uniqueness; you should see a green check mark appear on the right. (We will change this later to be your actual custom domain.)
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-front-door3.png)
+
+1. Next, we create a backend pool that contains just the SparkPost tracking domain. In Backend pools, select + to open Add a backend pool. Give this a name.
+
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-front-door4.png)
+
+1. Select "Add a backend". Set the backend host type to be "Custom host". Set the backend host name to be the correct endpoint address for your service, see [here](#endpoints).
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/azure-front-door5.png)
+
+    The backend host header field will be automatically filled in for you. Leave the HTTP port and HTTPS port settings at defaults. Click Add.
+
+    You can optionally configure a Health probe; there is no real need for this, as there is only one backend. If you do configure it, use `\f\a` as the path, and set the probe method to  `Get`, so that SparkPost will respond with a 3xx response (`Head` will get a 4xx response).
```

Comment:
oh man do we want people to do this? I'd rather not recommend people doing health checks directly against our API.

Reply:
No worries, I'll take it out.